notebooks/basics/quantum-circuits.ipynb

```diff
@@ -22,30 +22,43 @@
+    }
+   },
    "cell_type": "markdown",
    "id": "adfa4cb5",
    "metadata": {},
    "source": [
     "### 1.1 Boolean circuits\n",
     "\n",
-    "Here is an example of a (classical) Boolean circuit, where the wires carry binary values and the gates represent Boolean logic operations:\n",
+    "To illustrate these ideas, let's begin with a simple example. A *Boolean circuit* is one in which the output is a single classical bit (though the input could be $n$ bits). Here is an example:\n",
```

Comment:
```suggestion
    "To illustrate these ideas, let's begin with a simple example. A *Boolean circuit* is one in which the output is a single classical bit (though the input could be many bits). Here is an example:\n",
```

Reply:
Thanks Frank, I agree with all of these and made the change!